.gitignore

```diff
@@ -61,3 +62,5 @@ build/
 
 ## OS Specific
 .DS_Store
+
+desktop/.classpath
```

Comment:
Warning for no newline at the end of file. Unsure of the ramifications from this.